examples/httpServer.go

```diff
@@ -10,7 +10,8 @@ import (
 	"os"
 
 	//"github.com/pkg/profile"
```

Comment:
I know you didn't add this line. Not sure what it's doing there so if you fancy deleting it while you're at it that'd be great. Thanks.

Reply:
Don't think I added this. I'll remove it.